src/windows_registry.rs

```diff
@@ -15,6 +15,13 @@ use std::process::Command;
 
 use Tool;
 
+macro_rules! otry {
```

Comment:
So, when are we going to finally get a standard way of doing this in Rust/libstd? :stuck_out_tongue: